File: standing-out/StandingOut.Data/DTO/StripePlan.cs
using StandingOut.Data.Enums;
using StandingOut.Data.Models;
using System;
using System.ComponentModel.DataAnnotations;

namespace StandingOut.Data.DTO
{
    public class StripePlan
    {
        public StripePlan()
        {
        }

        public Guid StripePlanId { get; set; }

        public StripePlanType StripePlanType { get; set; }
        public StripePlanLevel StripePlanLevel { get; set; }

        [Required]
        [StringLength(1000)]
        public string StripeProductId { get; set; }
        [Required]
        [StringLength(250)]
        public string Description { get; set; }        
        public Subscription Subscription { get; set; }        
        public int FreeDays { get; set; }        
    }



}

File: standing-out/StandingOutStore/AngularSource/app/models/stripe-plan.ts
import {Subscription} from '../models/index';

export class StripePlan {
    stripePlanId: string;
    stripePlanType: number;
    stripePlanLevel: number;
    stripeProductId: string;
    description: string;
    subscription: Subscription;
    subscriptionId: string;
    freeDays: number;
}